src/main/java/org/jitsi/impl/protocol/xmpp/ChatRoomImpl.java

```diff
@@ -380,7 +415,7 @@ private void resetRoleForOccupant(EntityFullJid occupantJid)
         }
         else
         {
-            ChatMemberImpl member = members.get(occupantJid);
+            ChatMemberImpl member = getChatMember(occupantJid);
```

Comment:
Does this need this change? It seems like the only difference between the two methods is what happens if `occupantJid` is null, which already can't be true?

Reply:
I changed it because I was trying to fix spotbugs "inconsistent synchronization" warnings, but it's not needed. I reverted it.